keps/sig-windows/1981-windows-privileged-container-support/README.md

```diff
@@ -478,46 +487,84 @@ Because Windows privileged containers will work much differently than Linux priv
+
+A recording of the behavior differences from a SIG-Windows community meeting can be found [here](https://youtu.be/8GeZKXgvkdY?t=309).
+Note -  In the recording it was mentioned that this functionality might not be supported on WS2019.
+    This functionality will be available in WS2019 but will require an OS patch (ETA: July 2022).
+
+Additionally the following will be true for either volume mount behavior:
+
+- Named Pipe mounts will **not** be supported.
+    Instead named pipes should be accessed via their path on the host (\\\\.\\pipe\\*).
+    The following error will be returned if `hostProcess` containers attempt to use name pipe mounts -
+    https://github.com/microsoft/hcsshim/blob/358f05d43d423310a265d006700ee81eb85725ed/internal/jobcontainers/mounts.go#L40.
+- Unix domain sockets mounts also not not be supported for `hostProces` containers.
+    Unix domain sockets can be accessed via their paths on the host like named pipes.
+- Mounting directories from the host OS into `hostProcess` containers will work just like with normal containers but this is not recommend.
+    Instead workloads should access the host OS's file-system as if not being run in a container.
+  - All other volume types supported for normal containers on Windows will work with `hostProcess` containers.
+- `HostProcess` Containers will have full access to the host file-system (unless restricted by filed-based ACLs and the run_as_username used to start the container).
+- There will be no `chroot` equivalent.
+
+##### Compatibility
+
+During the alpha/beta implementations of this feature only **Symlink** volume mount behavior was implemented.
+This implemention did unlock a lot of critical use cases for managing Windows nodes in Kubernets clusters but did have some usability issues
+(such as https://pkg.go.dev/k8s.io/client-go/rest#InClusterConfig not working as expected).
+
+The **bind mount** volume mount behavior gives full access to the host OS's filesystem (an explicit goal of this enhancement) and addreses the usability issues with the initial approach.
+This approach requires the use of Windows OS APIs that were not present in Windows Server 2019 during alpha/beta implmentations of this feature.
+These APIs *will* be available in WS2019 beginning in July 2022 with the monthly OS security patches.
+Containerd v1.7+ will be required for this behavior.
+
+- On containerd v1.6 **symlink** volume mount behavior will always be used.
+- On containerd v1.7 **bind** volume mount behavior will always be used.
+  - If users are running nodes with Windows Server 2019 with security patches older than July 2022 the volume mounts for HostProcessContainers will fail.
+
+Users who have workloads that depend on the **symlink** mount behavior (ex: are expecting to find mounted volumes under `$CONTAINER_SANDBOX_MOUNT_POINT`) will need to stay on containerd v1.6 releases until their workloads are updated to be compatible with **bind** mount behavior.
```

Comment:
> workloads that depend on the **symlink** mount behavior (ex: are expecting to find mounted volumes under `$CONTAINER_SANDBOX_MOUNT_POINT`) will need to stay on containerd v1.6 releases until their workloads are updated to be compatible with **bind** mount behavior

This positioning is questionable for a GA feature. Remaining in beta until bind mount is available, then saying GA only supports bind mount is more what I would have expected

Reply:
We can keep the feature in `beta` for one more release and then go to stable after we containerd v1.7 releases and is deemed stable if that makes more sense. 

Either way we'll need to document that a breaking change will occur when updating from containerd v1.6 to v1.7 w.r.t. volume mount behavior.